psa-crypto/tests/mod.rs

```diff
@@ -78,9 +78,45 @@ fn import_integration_test() {
     }
 }
 
+#[test]
+fn export_key_pair_test() {
```

Comment:
Would it be possible to have a test where you put a previously generated key file in the right place and export that key? It would help with more accurate testing (instead of just making sure you get _something_ on the other side). Would also help with testing in Parsec, as we won't have to be that thorough when testing there.

Reply:
I guess another option is to import and export a "static" key (i.e. a byte vector).

Reply:
You mean like has been done in Parsec? Yes I can add that test here. I wasn't sure what kind of/how many tests are added to `psa-crypto` as there are only `generate` and `import` tests there currently.

Reply:
Yes, I think it makes more sense to put the tests at the lowest level where they apply - then neither Parsec nor other users of the crate need to test for that

Reply:
Ah ok, good point. I've used the PK from Parsec's test, imported it, exported it, and verified the original an exported values match.